src/ftest.jl

```diff
@@ -14,8 +14,10 @@ mutable struct FTestResult{N}
     pval::NTuple{N, Float64}
 end
 
+@deprecate issubmodel(mod1::LinPredModel, mod2::LinPredModel; atol::Real=0.0) StatsModels.isnested(mod1, mod2; atol=atol)
+
 """A helper function to determine if mod1 is nested in mod2"""
```

Comment:
should this docstring go and just lean on the `StatsModels` stub docstring?

Reply:
the only reason not to do that would be if this explaind the specific algorithm that's used here but it doesn't so we can add that later if needed.